BlueShift-iOS-SDK/InApps/BlueShiftNotificationSlideBannerViewController.m

```diff
@@ -308,6 +308,19 @@ - (UILabel *)createActionButtonLabel {
     return actionButtonlabel;
 }
 
+- (void)createBottomSafeAreaView {
+    CGRect frame = slideBannerView.frame;
+    frame.size.height = [self getBottomSafeAreaHeight];
+    frame.origin.y = frame.origin.y;
```

Comment:
Check if this can be optimized by adding if condition, checking safe area height is greater than zero. If its zero, we don't need to add the subview.